client/tailwind.config.js

```diff
@@ -26,6 +31,14 @@ module.exports = {
           position: 'absolute',
           'z-index': 10,
         },
+        '.city-skyline': {
+          background:
+            "url('https://static.vecteezy.com/system/resources/previews/008/045/334/original/silhouette-of-the-city-city-skyline-silhouette-modern-cityscape-for-t-shirt-abstract-city-landscape-illustration-free-vector.jpg')",
```

Comment:
Not the biggest fan of the background image, thought my screen was glitching at first 😅. Really like the colors on hover though 🤩

Reply:
hmm I see. any suggestions for picture themes? just the og background seemed a bit plain